projects/kubernetes-sigs/etcdadm/patches/0014-Add-retry-on-listing-members-in-join-command.patch

```diff
@@ -0,0 +1,90 @@
+From 35a9d8d692c24b89be947ba216980676670668fa Mon Sep 17 00:00:00 2001
+From: Wonkun Kim <wonkun@amazon.com>
+Date: Fri, 3 Jun 2022 15:33:17 -0500
+Subject: [PATCH] Update join.go
```

Comment:
Is it possible to change the commit message so it describes what it is, similar to the name of the patch file?

Reply:
Fixed.